### DISCUSSION TITLE
500 server error - htaccess option not allowed here

### DISCUSSION THREAD

**Discussion by ManySolutions** [firefly-iii/firefly-iii#6473] [Firefly III]
I have installed via softaculous but am unable to access the urls due to this error. In the plesk log it says:

core:alert] /var/www/vhosts/fire.domain.co.uk/httpdocs/firefly/.htaccess: Option All not allowed here

I have tried deleting and trying with various directories with no luck. the .htaccess file that the error links to is as follows:

> Optional: force HTTPS:
> <IfModule mod_rewrite.c>
>	RewriteEngine On
>	RewriteCond %{HTTPS} off
>	RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
> </IfModule>
 >To hide directory listing
>Options All -Indexes
 >To prevent access to .env and other files
><Files .*
 >Apache 2.2
><IfModule !mod_authz_core.c>
>Order deny,allow
>Deny from all
></IfModule
 >Apache 2.4
><IfModule mod_authz_core.c>
>Require all denied
></IfModule
></Files
><IfModule mod_alias.c
>RedirectMatch 301 ^/firefly/$ http://fire.domain.co.uk/firefly/public
></IfModule`

I can delete EVERYTHING else in .htaccess and the error does not change. EXCEPT when I change
` To hide directory listing Options All -Indexes`

When I delete this the error page changes to:

`FireflyIII 404 Not Found :(
404 — Firefly III cannot find this page.
The page you have requested does not exist. Please check that you have not entered the wrong URL. Did you make a typo perhaps?

If you were redirected to this page automatically, please accept my apologies. There is a mention of this error in your log files and I would be grateful if you sent me the error to me.`

Then in the plesk log, there are no errors.

The softaculous details when I go to edit:

`directory /var/www/vhosts/fire.domain.co.uk/httpdocs/firefly
URL fire.domain.co.uk/firefly
`
Feels like something is wrong here because of the 'firefly' on the end.

The installation message is:

`Firefly III has been successfully installed at :
http://fire.domain.co.uk/firefly
Administrative URL : http://fire.domain.co.uk/firefly/public/login`

and the document root in plesk is:
httpdocs/firefly/public

all of the directories just look messed up to me, but they were all automatic, i dont even know how to edit them?

I have searched and seen a post about the root needed to be 'public' folder, but from my end, I have done everything to make sure of that!

**Comment by JC5** [comment]
> Option All not allowed here

This is something your provider has to fix I'm afraid.

**Reply by ManySolutions** [reply]
I have full access to the server, how would I go about fixing this? I tried adding 

<Directory "/var/www/vhosts/fire.domain.co.uk">
	Options All
	AllowOverride All
	Order allow,deny
	Allow from all
</Directory>

as 'additional directives for HTTP (and s) under 'additional Apache directives' option in plesk to no affect.

So I assume I have to go under the hood with ubuntu?

I was just worried about tinkering just in case i make a global change and destroy everything.

**Reply by JC5** [reply]
> I have full access to the server, how would I go about fixing this?

It feels that your server doesn't want to accept /public as the root directory, and perhaps somewhere in the Apache config itself there's a rule that stops you from changing the AllowOverride. But to be honest, I'm not that good with Apache.

**Reply by ManySolutions** [reply]
as far as i can tell i need to edit the .conf file in apache2/sites-available but that is not where the conf. files are with softaculous. I am not sure where to look for them.

**Comment by ManySolutions** [comment]
Resolved
To enable firefly-iii installed via softaculous for my 500 server error
edit /etc/apache2
add 
<directory /var/www/etc/firefly-iii/>
OverrideAllow All
</directory>

I am now in.

**Reply by JC5** [reply]
Ahhhh that was it! Thanks for sharing!
